docs/en-US/references/tideways.md

```diff
@@ -9,44 +9,67 @@ VVV has different utilities available for developers, one of them is [Tideways](
 
 > Tideways saves you time by taking the guesswork out of your app's backend performance. Gain detailed insights, spot performance bottlenecks, and get real-time error detection alerts.
 
-## Suite
+## What Is It?
 
 Tideways is a PHP module for PHP 7.0+ that enables profiling everything that happens in a page.
 One must execute specific PHP code to use Tideways, this code gathers data and visualizes it.
 We added [XHGui](https://github.com/perftools/xhgui),a graphical interface for Tideways, that provides a complete ability to analyze the gathered data.
 
-## How to enable it
+## Installing Tideways
 
-XHGui needs a php file that is prepended to all PHP files (by a global php config) to turn on the profiling and save this information.
```

Comment:
I think that is important to mention that part

Reply:
It's only there because it would cause issues with XDebug but we've stated Tideways and XDebug can't be on at the same time anyway, so what are the chances someone is going to connect a debugger and run into that situation

Reply:
Well i think that as documentation we need to inform the user that we are doing that. usually with xdebug and other tools this prepending is not happening and maybe on some edge cases can create issues.